brainwidemap/encoding/pipelines/03_gather_results_pseudosessions.py

```diff
@@ -0,0 +1,183 @@
+# Standard library
+from functools import cache
+
+# Third party libraries
+import numpy as np
+import pandas as pd
+
+# IBL libraries
+from ibllib.atlas import BrainRegions
+from iblutil.numerical import ismember
+
+# Brainwidemap repo imports
+from brainwidemap.encoding.utils import get_id, remap
+
+
+def colrename(cname, suffix):
```

Comment:
Lots of code duplications here, move to utils module or other, also see comments in utils, some can be replaced by common IBL functions